guidelines/terms/22/target-offset.html

```diff
@@ -1,5 +1,5 @@
 <dt class="new"><dfn data-lt="target offsets">target offset</dfn></dt>
 <dd class="new">
 	<p class="change">New</p>	
-   <p>the distance measured from the farthest point of a target to the nearest point of the second target. Offset includes the target and spacing around the target. The target offset from A to B may be different then the offset from B to A, if the size of these targets differ.</p>
+   <p>the distance measured from the farthest point of a target (A), to the nearest point of the second target (B). The offset includes the target and spacing around the target. Targets other than A and B to not contribute to the spacing. The offset from A to B may be different then the offset from B to A, if the size of these targets differs.</p>
```

Comment:
typo `to not contribute` > `do not contribute` ?

Reply:
typo `different then the offset` > `different than the offset`

Reply:
addressed